README.md

```diff
@@ -57,6 +57,7 @@ There are several steps done in a pipeline:
        bugs: {
          url: "https://github.com/username/repo/issues",
        },
+       scripts: {},
```

Comment:
I think this should go in lib/package_json.ts and be automatically added when building the package.json if the scripts property isn't set? That way we don't need to document it for anyone.

Reply:
Sounds good, I'll prepare the change

Reply:
changed in df01664